libs/shared/ui/src/lib/components/icon/icons-status/canceling.tsx

```diff
@@ -0,0 +1,30 @@
+import { type ElementRef, forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { type IconSVGProps } from '../icon'
+import SpinnerIcon from './spinner'
+
+export const CancelingIcon = forwardRef<ElementRef<'div'>, IconSVGProps>(function CancelingIcon(
```

Comment:
```suggestion
export const CancelingIcon = forwardRef<ElementRef<typeof SpinnerIcon>, IconSVGProps>(function CancelingIcon(
```

Reply:
Already try but I can't do that 

<img width="815" alt="Capture d’écran 2023-09-19 à 14 26 00" src="https://github.com/Qovery/console/assets/533928/bb9b0ffa-1639-4a86-aee9-2a5b32d88b66">